src/ShadowDOM/wrappers/ShadowRoot.js

```diff
@@ -80,8 +80,13 @@
 
       var activeElement = wrap(unwrappedActiveElement);
 
-      // Loop while activeElement is not a shallow child of this ShadowRoot.
-      while (!this.contains(activeElement)) {
+      if (activeElement === this.host) {
```

Comment:
Please add a comment about how the host will not be considered the "activeElement" of a shadowroot.